### PR TITLE
fix: fix main program example functionality

### PR DESCRIPTION
**Pull request by smartinov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,44 +40,49 @@ Here is an example of how to use this package:
 package main
 
 import (
-        "log"
+	"log"
 
-        "github.com/mongo-go/lock"
+	"github.com/smartinov/mongo-lock/lock"
```
Comment (michaelfinch) [inline README.md]: Fix this and I'll merge it

Comment (smartinov) [reply README.md]: should be set to the previous value now "github.com/mongo-go/lock"

**Comment by smartinov** [comment]
Fix the offending line of code!

Was also wondering when you do a master commit, to do some kind of a wrapper with a timeout which would fulfill the contract from sync.RWMutex or Mutex so that it can be a drop-in replacement for the standard mutex?

**Comment by michaelfinch** [comment]
Thanks! I'll go ahead and merge this.

I'll take a look at that to see if it makes sense to include in this package. In the meantime, have you seen https://github.com/Clever/mongo-lock-go?

**Comment by smartinov** [comment]
@michaelfinch honestly, I've been there but i didn't see a good mechanic for having stale locks, if something goes wrong. Like if a routine died without deferring cleanup of the lock. Also like the metadata information.

Also the possibility of automatically ensuring the  indices are there on the collection by using sync.Once and so on, which i implemented in a custom wrapper.

So it might be a nice extension on that!

**Comment by michaelfinch** [comment]
Filed https://github.com/mongo-go/lock/issues/3 to track

**Pull request merged**